SignUp.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import TextBox from "./textBox"
+import Button from "./button"
+import "./styles.css"
+
+
+var SignUp=()=> {
+  return (
+    <div class ="Text">
+    <center>
+    <form id="form1">
+    <h1>SignUp Form</h1>
+    <textBox

```

Comment:
does this code running on machine ?? 
you imported it as `import TextBox from "./textBox"` but using it as `textBox`